Src/BootCamp.Chapter/TimeCommand.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace BootCamp.Chapter
+{
+    internal class TimeCommand
+    {
+        private static readonly TimeSpan OneDay = new TimeSpan(24, 0, 0);
+
+        private FileInfo _outputFile;
+        private readonly TimeSpan _beginning;
+        private readonly TimeSpan _end;
+
+        public TimeCommand(FileInfo outputFile, string argument)
```

Comment:
Consider modifying outputFile to a Func<Stream> to be invoked when it's time to write. Removes dependency on file syetem.